db/migrate/0_init_tables.rb

```diff
@@ -14,16 +14,13 @@ def change
     create_table "group".pluralize.to_sym, id: :serial do |t|
       t.string :name
       t.string :email
```

Comment:
Consider a column called contact_method
Consider a column called address which is a jsonb type. The content of the field has to conform the json schema defined by the contact method